### ISSUE TITLE
Custom exports not working

### ISSUE THREAD

**Issue by melTr** [JabRef/jabref#4036]
- [ x] I have tested the latest master version from http://builds.jabref.org/master/ and the problem persists

Hi,
I used the same export files since many years, but in this version, I can't export. My file is name "abstract.layout". I tried to rename to "abstract", but didn't work either. 
I tried different exports, and for all of them I get this same error : 

```
Error exporting to clipboard
java.io.IOException: Cannot find layout file: '/home/mel/Documents/jabref/abstract/abstract.layout.layout'.
	at org.jabref.logic.exporter.TemplateExporter.getReader(TemplateExporter.java:184)
	at org.jabref.logic.exporter.TemplateExporter.export(TemplateExporter.java:248)
	at org.jabref.gui.exporter.ExportToClipboardAction.run(ExportToClipboardAction.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at spin.Invocation.evaluate(Invocation.java:175)
	at spin.off.SpinOffEvaluator$1.run(SpinOffEvaluator.java:108)
	at java.lang.Thread.run(Thread.java:748)
```
JabRef 4.3-dev--snapshot--2018-05-12--master--899d267bb
Linux 4.4.0-124-generic amd64 
Java 1.8.0_131

Thanks!

**Comment by Siedlerchr** [comment]
Hi,
this is probably related to #4013  and will be fixed with #4015 
In the meantime you could try out this version: https://builds.jabref.org/fixCustomExportFileType/

**Comment by drmikewest** [comment]
Works for me, as a user (wndows10). 
Tested JabRef_windows-x64_4_3-dev--snapshot--2018-05-15--fixCustomExportFileType--289c439ea.exe
and custom exports function again as they did prior to v4.1, all looks Ok on from this view.

**Comment by Siedlerchr** [comment]
Closed by #4015 
Please try the latest version and make sure you have a backup of your database.
https://builds.jabref.org/master/
